src/main/java/com/minecolonies/commands/ColonyCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * Minecolonies root command.
+ * <p>
+ * Manages all sub commands.
+ */
+public class ColonyCommand extends AbstractSplitCommand
+{
+
+    public static final String DESC = "colony";
+
+    private final ImmutableMap<String, ISubCommand> subCommands =
+      new ImmutableMap.Builder<String, ISubCommand>()
+        .put(KillCitizen.DESC, new KillCitizen(MinecoloniesCommand.DESC, ColonyCommand.DESC, KillCitizen.DESC))
```

Comment:
General question about commands. Why do we pass the command name to itself. KillCitizen already knows what its command string is, so why do we have that as a parameter in the constructor.